.github/workflows/build_centos7.yml

```diff
@@ -0,0 +1,57 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+  workflow_dispatch:
+
+jobs:
+  build_centos:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        container: ["centos:7"]
+
+    container:
+      image: ${{ matrix.container }}
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install make and sudo
+        run: |
+          yum update -y
+          yum install -y sudo
+          yum groupinstall -y "Development Tools"
+          yum install -y  gtk3 libXext libXrender libXtst libX11 libXau libXcursor libXdmcp libXfixes libXi libXmu libXp libXpm libXt
+
+      - name: Install requirements
+        run: |
+          make deps
+          make venv
+          . venv/bin/activate
+          export PATH=`npm bin`:$PATH
+          make pythondeps
+          python -c "import setuptools; print(setuptools.__version__)"
+          pip --version
+
+      - name: Install Firefox
+        run: |
+          cd /usr/local
+          curl -O http://ftp.mozilla.org/pub/firefox/releases/75.0/linux-x86_64/en-US/firefox-75.0.tar.bz2
+          tar xvjf firefox-75.0.tar.bz2
+          sudo ln -s /usr/local/firefox/firefox /usr/bin/firefox
+
+      - name: Install Simphony-Remote
+        run: |
+          . venv/bin/activate
+          make install
+
+      - name: Install development requirements
+        run: |
+          . venv/bin/activate
+          make devdeps
```

Comment:
Yes, the `pythontest` suite runs successfully, while `jstest` failed because of some discrepancy in PhantomJS and/or Node, although I couldn't find any difference between the versions used on Ubuntu and CentOS, and I didn't want to spend too much time on that. I can re-add the python tests, and leave the `jstest` investigation for a later time if you agree.

Reply:
I see - let's just include the `pythontest` suite for now then. If we could also add a follow up issue to include JS tests on the CentOS builds, that would be great.